src/common/IPC/README.md

```diff
@@ -0,0 +1,2 @@
+This directory contains code relating to Inter-Process Communication (i.e. between the engine and a
```

Comment:
Why not removing the repetitive and obvious `This directory contains`? It looks like all directories are directories containing something, so maybe we can just start with what it contains, this way:

```
Code relating to Inter-Process Communication (i.e. between the engine and a gamelogic VM).
```

Reply:
I don't consider it so obvious as to go without saying. Plus, it changes the meaning. `Code for the engine` suggests the *entire* source of the engine, while `This directory contains code for the engine` makes it clearer that it doesn't necessarily mean all engine code.

Reply:
OK